exo-go/test_helpers/add_feature_context.go

```diff
@@ -28,7 +28,7 @@ func createEmptyApp(appName, cwd string) error {
 	fields := []string{"AppName", "AppDescription", "AppVersion", "ExocomVersion"}
 	inputs := []string{appName, "Empty test application", "1.0.0", "0.22.1"}
 	for i, field := range fields {
-		if err := process.WaitForTextWithTimeout(field, 1000); err != nil {
+		if err := process.WaitForTextWithTimeout(field, 5000); err != nil {
```

Comment:
The 1000 millisecond timeout wasn't enough on travis. It shouldn't ever take 5 seconds but this is just to give it a nice buffer